exoctk/phase_constraint_overlap/phase_constraint_overlap.py

```diff
@@ -12,11 +23,24 @@ def calculate_phase(period, t0, obsDur, winSize):
     return minphase, maxphase
 
 def get_transit_details(target_name):
-    ''' This function will use the target name to get the transit 
-    details from exoplanets.org or somewhere.'''
+    '''send request to exomast restful api for target information.
+    '''
 
+    target_url = build_target_url(target_name)
+    r = requests.get(target_url)
+
+    if r.status_code == 200:
+        target_data = r.json()
+    else:
+        print('Whoops, no data for this target!')
 
+    # Requests from restapi can have multiple catalogs.
+    # Discussion about which one we should use should 
+    # be had in the future.
+    for item in target_data:
+        print(item['catalog_name'], item['orbital_period'])
 
+    # are t0, obsDur, and winSize available via ExoMAST api?
```

Comment:
Good question! I'll prep an email to Kevin and CC you so that we have a more in-depth explanation of our parameters and understand what the individual variables are.